spec/dummy/config/environments/test.rb

```diff
@@ -26,11 +26,6 @@
   # Disable request forgery protection in test environment.
   config.action_controller.allow_forgery_protection = false
 
-  # Tell Action Mailer not to deliver emails to the real world.
-  # The :test delivery method accumulates sent emails in the
-  # ActionMailer::Base.deliveries array.
-  config.action_mailer.delivery_method = :test
```

Comment:
This is questionable. We also don't want emails to be sent via notify either when running tests. What will happen to emails when that option gets removed?